prefix_finder/frontend/templates/list.html

```diff
@@ -51,13 +65,22 @@ <h2>Description</h2>
           </ul>
 
           <div class="single-meta-info__instructions">
-            <h3>To find identifiers</h3>
-            <p>{{ org_list.access.guidanceOnLocatingIds|urlize|linebreaks}}</p>
+            <h3>Search options</h3>
```

Comment:
can we swap line 68 and line 69? (so that 'Search options' appears only % if % there is guidance)

Reply:
Yep! https://github.com/OpenDataServices/org-ids/commit/aa5d9561b3a02ed9fb0105509ff390298794e61d